packages/terra-avatar/src/Avatar.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import TerraImage from 'terra-image';
+import styles from './Avatar.scss';
+
+const cx = classNames.bind(styles);
+
+const AvatarVariants = {
+  FACILITY: 'facility',
+  USER: 'user',
+};
+
+const propTypes = {
+  /**
+   * The text content that specifies the alternative text for the image.
+   */
+  alt: PropTypes.string,
+  /**
+   * The text content of the aria-label for accessibility.
+   */
+  ariaLabel: PropTypes.string,
```

Comment:
I am wondering if this should be required

Reply:
Brett and I talked. We decided to leave it as an optional string.

Reply:
There's been some discussion around this already. @JakeLaCombe and @bjankord decided it was best to default it to undefined rather than make it required.

Reply:
Essentially, it's not always easy to provide a value for this attribute, and may not always have a desire to be accessible. Thus, to ensure we can still keep it as accessible when it needs to be, we can make this optional.

Reply:
@JakeLaCombe, if that's the case, is there a reason that we don't just let it get spread onto the `div` below via `customProps`? I know that react accepts the first-class prop `ariaLabel` in general for it's components, so I'm not sure if it's a matter of using `aria-label` vs`ariaLabel` being an issue, but if we aren't using Avatar's `ariaLabel` prop, is there a reason to explicitly call it out here?

Reply:
I agree with @JakeLaCombe on recommending the first classing prop, as it sticks out more to the consuming teams.